apps/web/components/Organization/Wrapper/Themes/RikissaksoknariTheme/RikissaksoknariHeader.tsx

```diff
@@ -36,6 +38,14 @@ const RikissaksoknariHeader = ({ organizationPage }: HeaderProps) => {
 
   const screenWidth = getScreenWidthString(width)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'